quickwit-metastore/src/metastore/file_backed_metastore/index.rs

```diff
@@ -29,28 +30,83 @@ use crate::{IndexMetadata, MetastoreError, MetastoreResult, Split, SplitMetadata
 
 /// A `Index` object carries an index metadata and its split metadata.
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub(crate) struct Index {
```

Comment:
Necessary for writing the backward-compatibility test.

Reply:
Yeah that's a bummer but that makes sense.